index.js

```diff
@@ -154,10 +154,22 @@ function packAndInstallLibrary(name, dir, targetDir) {
     fs.mkdirSync(libDestDir, { recursive: true })
 
     const tmpName = name.replace(/[\s\/]/g, "-").replace(/@/g, "")
+
+   
+
     // npm replaces @... with at- where yarn just removes it, so we test for both files here
     const regex = new RegExp(`^(at-)?${tmpName}(.*).tgz$`)
 
-    const packagedName = fs.readdirSync(dir).find(file => regex.test(file))
+    let packagedName = fs.readdirSync(dir).find(file => regex.test(file))
+
+    if(hasYarn) {
```

Comment:
According to the [docs](https://github.com/camacho/yarn-or-npm#module), `hasYarn` is a function.

Have you tested your code on yarn 1 or 2 with these changes?